conda/recipes/libkvikio/meta.yaml

```diff
@@ -72,8 +74,10 @@ outputs:
     requirements:
       build:
         - cmake {{ cmake_version }}
+        - aws-sdk-cpp
```

Comment:
```suggestion
```
@KyleFromNVIDIA do we `aws-sdk-cpp` here? Asking because `libcufile-dev` isn't need?